src/sgame/sg_bot_util.cpp

```diff
@@ -1198,15 +1198,20 @@ void BotGetIdealAimLocation( gentity_t *self, botTarget_t target, vec3_t aimLoca
 {
 	//get the position of the target
 	BotGetTargetPos( target, aimLocation );
+	bool target_buildable = BotGetTargetType( target ) == entityType_t::ET_BUILDABLE;
+	//this retrieves the target's species, to aim at weak point:
+	// * for humans, it's the head (but code only applies an offset here, with the hope it's the head)
+	// * for aliens, there is no weak point, and human bots try to take missile's speed into consideration (for luci)
+	team_t target_team = BotGetTargetTeam( target );
 
-	if ( BotGetTargetType( target ) != entityType_t::ET_BUILDABLE && BotTargetIsEntity( target ) && BotGetTargetTeam( target ) == TEAM_HUMANS )
+	if ( target_buildable == false && BotTargetIsEntity( target ) && target_team == TEAM_HUMANS )
 	{
 
 		//aim at head
 		aimLocation[2] += target.ent->r.maxs[2] * 0.85;
 
 	}
-	else if ( BotGetTargetType( target ) == entityType_t::ET_BUILDABLE || BotGetTargetTeam( target ) == TEAM_ALIENS )
+	else if ( target_buildable == true || target_team == TEAM_ALIENS )
```

Comment:
```suggestion
	else if ( target_buildable || target_team == TEAM_ALIENS )
```

But I'm really wondering if that one shouldn't be 
```suggestion
	else if ( !target_buildable && target_team == TEAM_ALIENS )
```

Reply:
I don't know the code enough yet, so I tried to change nothing in this set (but yes, it's very tempting to integrate some changes in there)